make_cert.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
So this script should run in case we are not expecting self-sign certificate. right ? In that case we need to update the `smoke-test.sh` accordingly

Reply:
I don't know if this script should be run for anything real.... its what I was using to run locally.  I imagine for a real deployment, an actual signed & trusted cert should be used.